### PR TITLE
Update README.md, python example

### PR DESCRIPTION
**Pull request by jeanlauliac**
### 🤔 What's changed?

Add the uri field which is required by the current implementation of `Compiler`.

### 🏷️ What kind of change is this?

- :bank: Refactoring/debt/DX (improvement to code design, tooling, documentation etc. without changing behaviour)

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [ ] I've changed the behaviour of the code
  - [ ] I have added/updated tests to cover my changes.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] Users should know about my change
  - [ ] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../blob/main/CHANGELOG.md), linking to this pull request.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/PULL_REQUEST_TEMPLATE.md)*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ from gherkin.pickles.compiler import Compiler
 
 parser = Parser()
 gherkin_document = parser.parse("Feature: ...")
+gherkin_document["uri"] = "uri_of_the_feature.feature"
```
Comment (olleolleolle) [inline README.md:131]: 🔍 Looks good! Thanks!

Minor question, not a blocker: Is `"uri_of_the_feature.feature"` an acceptable "uri", here?

- If yes, that's cool,
- If not, perhaps we can use something that would be a real URI, (e.g. `"file:///home/alice/projects/my_app/uri_of_the_feature.feature"`, `"http://example.com/gherkin_features/uri_of_the_feature.feature"`)

(Minor question #2: can Parser.parse dig out "uri" out of given text to parse? Sounds improbable.)

**Review by olleolleolle — COMMENTED** [review]
I asked a question, to understand a little more. (I don't know the Compiler interface!)

File: README.md (hunk 1 above)
Comment (jeanlauliac) [reply README.md:131]: I copied this URL from another language's example. We can pick an example url and change it everywhere.

Comment (olleolleolle) [reply README.md:131]: No need to, change, let's merge this improvement as it stands , thanks for filling me in.

**Comment by aslakhellesoy** [comment]
Hi @jeanlauliac,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**